docs/platform/howto/manage-project.md

```diff
@@ -3,61 +3,52 @@ title: Manage projects
 ---
 
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons"
 
 ## Create a project
 
-In the [Aiven Console](https://console.aiven.io/), follow these steps to
-create a project:
-
-1.  Click **Project** and select **Create project**.
+1.  Click **Projects** and select **Create project**.
 2.  Enter a name for the project.
 3.  Select an organization or organizational unit to add the project to.
-4.  Select a
-    [billing group](/docs/platform/concepts/billing-groups). The costs from all services within this project will be
-    charted to the payment method for that billing group.
+4.  Select a [billing group](/docs/platform/concepts/billing-groups).
+    The costs from all services in this project are charged to the
+    payment method for this billing group.
 
 ## Rename a project
 
 :::important
-
-- Except for Aiven for Apache Kafka®, all services have backups that
-  are restored when you power them back on.
-- The project name in your DNS records will not be updated.
-
+The project name in your DNS records will not be updated.
 :::
 
-To rename a project in the [Aiven Console](https://console.aiven.io/):
-
 1. Power off all services in the project.
-1. In the **Project**, click **Settings**.
-1. Edit the **Project name**.
+   :::note
+    Except for Aiven for Apache Kafka®, all services have backups that
+    are restored when you power them back on.
+   :::
+
+1. In the project, click <ConsoleLabel name="projectsettings"/>.
+1. In the **Project settings**, edit the **Project name**.
 1. Click **Save changes**.
 
 ## Move a project
 
-To move a project from one organizational unit to another in the [Aiven
-Console](https://console.aiven.io/):
+You can move a project to another organization or organizational unit.
+Users with admin access to projects can move them to another
+organizational unit or up a level to the organization.
 
-1.  Click **Admin** and select the organizational unit with the project
-    to move.
+To move a project to a different organization, you must be a super admin
```

Comment:
How about adding the link:

`[super admin](/docs/platform/howto/make-super-admin)`?